README.md

```diff
@@ -4,6 +4,17 @@
 
 This library provides a TileDB backend engine for xarray `open_dataset` function.
 
+Example usage:
+
+```python
+import xarray as xr
+dataset = xr.open_dataset(
+    "tiledb_array_uri",
+    backend_kwargs={"key": key, "timestamp": timestamp},
+    engine="tiledb"
+    )
+```
+
```

Comment:
(suggest to add install steps here, maybe after release on pypi -- since this will likely be the first search result for tiledb-xarray)

Reply:
Will do. Unfortunately, we can't release to PyPI until xarray releases their 0.18 version.